core/src/test/java/org/apache/cxf/helpers/FileUtilsTest.java

```diff
@@ -77,6 +78,8 @@ public void testGetFiles() throws URISyntaxException {
 
         List<File> foundFiles2 = FileUtils.getFiles(directory, ".*\\.class$");
 
+        Collections.sort(foundFiles);
```

Comment:
@prathyushreddylpr thank you for the fix, although I've never seen this test failing, the File System API indeed does not guarantee an order. There is an issue with this patch, the `Collections.sort` requires the collection to be modifiable which we cannot guarantee.

Reply:
Thanks @reta for the feedback. I have changed the code to check the contents of two lists in a better way which would ignore the order of the elements in the list.